bin/casher

```diff
@@ -130,10 +130,10 @@ class Casher
       return
     end
 
-    msg "changes detected, packing new archive"
-
     @push_tar  = File.expand_path('push.tgz', @casher_dir) if path_ext(url) == 'tgz'
 
+    msg "changes detected, packing new archive: #{@push_tar}"
```

Comment:
`@push_tar` is going to be a simple `push.tbz`, and it'll be uninteresting.

If you are interested in the name, you'd want to display parts of `url`.

Reply:
@BanzaiMan Should I just change `@push_tar` for `url`? e.g.

```ruby
msg "changes detected, packing new archive: #{url}"
```

Thanks!

Reply:
`url` would include the signed URL for `POST`ing the archive, so we should not reveal it. We'll have to drop the request parameters (everything after the last `&`), as well as the parts leading up to (and including) the penultimate `/`.